Testing/VelaptorTests/Fakes/WindowFake.cs

```diff
@@ -17,9 +18,10 @@ public class WindowFake : Window
     /// </summary>
     /// <param name="window">Mocked window..</param>
```

Comment:
Remove add'l period at the end of the sentence.
```suggestion
    /// <param name="window">Mocked window.</param>
```